android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/ConfigurationRegistry.kt

```diff
@@ -176,6 +171,14 @@ constructor(
         .readText()
         .decodeResourceFromString<Composition>()
         .run {
+          section.first { it.title == "ic_households" }.let {
+            Timber.d("FIKRI section ${it.focus.reference}")
+            val resource = fhirResourceDataSource.loadData("Binary?_id=${it.focus.reference.extractLogicalIdUuid()}")
+            resource.entry.forEach { entryComp ->
+              Timber.d("FIKRI RES ${entryComp.resource.id}")
```

Comment:
Can we improve this log if it is necessary

Reply:
Sure